packages/graphql-migrations/src/plugin/SchemaTransformer.ts

```diff
@@ -0,0 +1,27 @@
+import { GraphQLSchema } from 'graphql';
+import { filterNonModelTypes } from '@graphback/core'
+import { SchemaComposer } from 'graphql-compose'
+
+export interface SchemaTransformerPlugin {
+  transform(schema: GraphQLSchema): GraphQLSchema
+}
+
+export const removeNonModelTypesFromSchema: SchemaTransformerPlugin = {
```

Comment:
This implementation will not work, so need to change it

Reply:
Seems like separate issue to me ;D

Reply:
https://github.com/aerogear/graphback/pull/1211#discussion_r417421475

Related to this